src/views/Sales/SaleDetails.vue

```diff
@@ -45,7 +49,11 @@ const currentSale = computed(() => store.getters["sales/getSaleById"](saleId));
 const saleProducts = currentSale.value.products.map((product) =>
   formatProducts(product)
 );
-const ownerColumn = computed(() => [store.state.products.tableColumnOwner]);
+const ownerColumn = computed(() => [
```

Comment:
Last thing: this is now `additionalColumns` (it's not jus the owner column)

Reply:
done